app/projects/views.py

```diff
@@ -462,16 +462,53 @@ def grant_access_with_view_permissions(request):
 
     return HttpResponse(200)
 
+
+def _create_agreement_form_list(project, user, current_step):
```

Comment:
Are these functions prefixed because theyre helper functions instead of views?

Reply:
Yeah, I think in Python usually one underscore denotes a "private" or internal use method. Not that these are classes yet. Maybe someday!